### PR TITLE
xorex: init at 0.3.0

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/xorex/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+python3.pkgs.buildPythonApplication rec {
```
Comment (SuperSandro2000) [inline pkgs/tools/security/xorex/default.nix:5]:
```suggestion
}:

python3.pkgs.buildPythonApplication rec {
```

File: pkgs/tools/security/xorex/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Neo23x0";
+    repo = "xorex";
+    rev = version;
+    sha256 = "rBsOSXWnHRhpLmq20XBuGx8gGBM8ouMyOISkbzUcvE4=";
+  };
+
+  format = "other";
```
Comment (SuperSandro2000) [inline pkgs/tools/security/xorex/default.nix:16]:
```suggestion
```

File: pkgs/tools/security/xorex/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "xorex";
+  version = "0.3.0";
```
Comment (SuperSandro2000) [inline pkgs/tools/security/xorex/default.nix:7]:
```suggestion
  version = "0.3.0";
  format = "other";
```

File: pkgs/tools/security/xorex/default.nix [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  };
+
+  format = "other";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    chmod +x xorex.py
+    mv xorex.py $out/bin
```
Comment (SuperSandro2000) [inline pkgs/tools/security/xorex/default.nix:23]:
```suggestion
    mv xorex.py $out/bin/xorex
```

**Comment by SuperSandro2000** [comment]
And PR number 120000. 🎉

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 120000` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>xorex</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Related to #81418

**Pull request merged**